.github/workflows/goreleaser-fleetctl-docker-deps.yaml

```diff
@@ -0,0 +1,58 @@
+# Builds and releases to production the fleetdm/bomutils:latest and fleetdm/wix:latest
+# docker images, which are the docker image dependencies of the fleetctl command.
+#
+# This is separate from Fleet releases because we only release
+# fleetdm/bomutils and fleetdm/wix only if we add new dependencies
+# or for security updates.
+name: Release fleetctl docker dependencies
+
+on:
+  push:
+    tags:
+      - "fleetctl-docker-deps-*"
```

Comment:
If we go this approach, we'll need this tag to be protected.

Reply:
@lukeheath

Reply:
I think @sharon-fdm can update protections as well.

Reply:
Added with @lucasmrod